Lib/trufont/windows/fontWindow.py

```diff
@@ -678,13 +684,40 @@ def copy(self):
             packGlyphs = (
                 glyphs[index] for index in sorted(
                     self.glyphCellView.selection()))
+
+        fontHeight = font.info.ascender - font.info.descender
+        glyphSet = {n: font[n] for n in font.glyphOrder}
```

Comment:
You can just pass the font instead of copying it.

Reply:
I keep forgetting that 😅